.github/workflows/push_pr.yml

```diff
@@ -25,19 +25,12 @@ jobs:
   compile_and_test:
     strategy:
       matrix:
-        flink: [ 1.17.1, 1.18.0 ]
-        jdk: [ 8, 11, 17 ]
-        exclude:
-          - jdk: 17
-            flink: 1.17.1
+        flink: [ 1.17.2 ]
+        jdk: [ '8, 11' ]
+        include:
+          - flink: 1.18.1
+            jdk: '8, 11, 17'
     uses: apache/flink-connector-shared-utils/.github/workflows/ci.yml@ci_utils
     with:
       flink_version: ${{ matrix.flink }}
-      jdk_version: ${{ matrix.jdk }}
-  python_test:
-    strategy:
-      matrix:
-        flink: [ 1.17.1, 1.18.0 ]
-    uses: apache/flink-connector-shared-utils/.github/workflows/python_ci.yml@ci_utils
-    with:
-      flink_version: ${{ matrix.flink }}
```

Comment:
should we remove ci for python part?

IIRC currently it is done separately

Reply:
Oh wow I completely missed the Python part. Let me check

Reply:
I think we should keep them in for PRs, and actually we should also have them for weekly runs.

Reply:
+1